### PR TITLE
Remove enzyme from uniforms

### PR DESCRIPTION
**Pull request by ErnestTeluk**
- Removed `enzyme` from uniforms repo

### CODE REVIEW

**Review by zaxoavoki — APPROVED** [review]

File: jest.config.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ module.exports = {
     '^uniforms([^/]*)(.*)$': '<rootDir>/packages/uniforms$1/src$2',
   },
   preset: 'ts-jest',
-  setupFiles: ['./scripts/setupEnzyme.ts', './scripts/setupFilterWarnings.ts'],
+  setupFiles: ['./scripts/setupRTS.ts', './scripts/setupFilterWarnings.ts'],
```
Comment (piotrpospiech) [inline jest.config.js:10]: What is `RTS`? That file looks like configuration needed for the `JSDOM` ([docs](https://jestjs.io/docs/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom)). Maybe use `JSDOM` in the filename.

Comment (ErnestTeluk) [reply jest.config.js:10]: RTS = React Testing Library

Comment (piotrpospiech) [reply jest.config.js:10]: I would say `RTL` is better than `RTS`, but still it isn't setting up React Testing Library, but Jest.

Comment (ErnestTeluk) [reply jest.config.js:10]: i change name for just` setupJest`

**Review by piotrpospiech — APPROVED** [review]

**Review by zaxoavoki — APPROVED** [review]

**Review by Monteth — APPROVED** [review]

**Pull request merged**